src/app/redux/UserSaga.js

```diff
@@ -372,13 +373,36 @@ function* usernamePasswordLogin2({
         // Does not need to be fatal
         console.error('Server Login Error', error);
     }
+
+    // Feature flags
+    yield fork(
```

Comment:
I have't used forks before, nor generators in the way `getFeatureFlags` appears to implement the pattern.

I have a strong appreciation for indicating this code is async, but is there a technical reason for us to have `getFeatureFlags` implemented as a classic function using `await` in place of `yield`?

Reply:
`fork` is a redux-saga concept (not the syscall fork)

generators are used in redux-saga to simplify flow control - rather than actually calling the async methods, a saga yields actions (the `put`s and `call`s and `forks`) -- redux-saga will handle pausing the generator function while those methods run async.